src/engraving/dom/select.cpp

```diff
@@ -537,6 +537,17 @@ void Selection::appendChord(Chord* chord)
     }
 }
 
+void Selection::appendTupletHierarchy(Tuplet* innermostTuplet)
+{
+    appendFiltered(innermostTuplet);
```

Comment:
Shouldn't we check `if (!mu::contains(m_el, innermostTuplet)`? Or is it only the first chord of a tuplet where `tuplet()` is set?

Reply:
That'd result in only the outer and the __first__ inner tuplet to get selected

Reply:
@cbjeukendrup is right, I've pushed this tweak.

I tried to repro the above behaviour @Jojo-Schmitz but couldn't (nor can I see why this might happen). If you find it's present in the latest iteration please let me know!

Reply:
I had that issue with my backport to 3.x

Reply:
Oh, I see now, between your code and @cbjeukendrup's there's  `!`